BaragonAgentService/src/main/java/com/hubspot/baragon/agent/config/BaragonAgentConfiguration.java

```diff
@@ -40,6 +40,9 @@
   @JsonProperty("hostname")
   private String hostname;
 
+  @JsonProperty("nginxPort")
```

Comment:
While has mostly been built around `nginx` we've tried to keep as much opinion out of baragon as possible. Maybe something like `loadBalancerPort` or `listenerPort` instead here?